apps/librepcb/firstrunwizard/firstrunwizardpage_workspacepath.ui

```diff
@@ -14,7 +14,7 @@
    <string>WizardPage</string>
   </property>
   <property name="title">
-   <string>Select Workspace Path</string>
+   <string>Select Wor&amp;kspace Path</string>
```

Comment:
I just noticed these changes now. Do we want them?

Reply:
Yes you can keep them, these are the automatically created keyboard shortcuts (ALT+k in this case). Although I wonder why this is created for the title of a wizard page... ;)